cmd/upgrade_test.go

```diff
@@ -100,21 +85,6 @@ var _ = Describe("the upgrade command", func() {
 					deploymentDirectory: "_output/mydir",
 					upgradeVersion:      "1.9.0",
 					location:            "southcentralus",
-					authArgs:            authArgs{},
-				},
-				expectedErr: fmt.Errorf("--subscription-id is required (and must be a valid UUID)"),
```

Comment:
Moved validateAuthArgs to load() so we don't check for subID in validate anymore